build.gradle

```diff
@@ -1,7 +1,14 @@
 buildscript {
     apply from: rootProject.file("dependencies.gradle")
     repositories {
-        google()
+        maven {
+            url 'https://dl.google.com/android/maven2/'
```

Comment:
doesn't google() exist?

Reply:
due to rocker and other breaking upgrades since we are on AGP 4.x, we need to set the property of the maven url to use pom instead of gradle metadata & we can't do this w/ `google`